src/patternfly/components/DataToolbar/docs/data-toolbar-toggle-group-intro.md

```diff
@@ -0,0 +1,3 @@
+### Toggle group modifier
+The `.pf-m-toggle-group` controls when, and at which breakpoint, filters will be hidden and revealed. By default, all filters are hidden until the specified breakpoint is reached.
+- `.pf-m-reveal-on-{md, lg, xl}`: controls when filters are revealed and when toggle button is hidden.
```

Comment:
bullet isn't showing up, this example doesn't have `className="is-layout-page"`. Some others are missing it, too, not sure if it should be added to those.